sdks/ruby/src/lib/hatchet/clients/rest.rb

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+# typed: strict
+
+require 'fileutils'
+require 'rbconfig'
+
+# Integration file for generated Hatchet REST API client
+# This file loads the generated REST client and makes it available under the Hatchet::Clients::Rest namespace
+
+begin
+  # Set up load paths for the generated client
+  rest_lib_path = File.expand_path("rest/lib", __dir__)
```

Comment:
will these paths also work when the library is packaged up? I'm not too familiar with this, but this seems a bit odd to me to be using file paths in one of the clients

Reply:
i think this can be removed.. i made some changes to the default loader.

Reply:
nice 👍